### DISCUSSION TITLE
Truffle Console not working

### DISCUSSION THREAD

**Discussion by Leptron-Technologies** [PavanAnanthSharma/FlashLoan-Using-Uniswap-MoneyLegos#6] [Q&A] [answered]
![download](https://user-images.githubusercontent.com/106593836/177320372-8893780c-0f56-4d77-9933-5c8efcb11cd8.png)

**Comment by PavanAnanthSharma** [comment, marked as answer]
Please make sure you have the latest version of npm and node installed. I had the same issue, I updated npm and node to latest version and it worked.

```npm install -g truffle``` works.
